sdk/keyvault/azure-security-keyvault-certificates/perf-tests.yml

```diff
@@ -14,4 +14,5 @@ Tests:
 - Test: get-certificate
   Class: GetCertificate
   Arguments:
-  - --parallel 64
+    - --warmup 0 --duration 1
```

Comment:
```suggestion
```

I would remove the `--warmup 0 --duration 1` unless you really want them.  I previously used these to run tests really quickly, but a single run of `--parallel 4` should be fast enough.